.coveragerc

```diff
@@ -1427,13 +1422,6 @@ omit =
     homeassistant/components/telegram_bot/__init__.py
     homeassistant/components/telegram_bot/polling.py
     homeassistant/components/telegram_bot/webhooks.py
-    homeassistant/components/tellduslive/__init__.py
-    homeassistant/components/tellduslive/binary_sensor.py
-    homeassistant/components/tellduslive/cover.py
-    homeassistant/components/tellduslive/entry.py
-    homeassistant/components/tellduslive/light.py
-    homeassistant/components/tellduslive/sensor.py
-    homeassistant/components/tellduslive/switch.py
```

Comment:
`tellduslive` has very low coverage... should it still qualify as `gold`?
```
---------- coverage: platform linux, python 3.12.2-final-0 -----------
Name                                                    Stmts   Miss  Cover   Missing
-------------------------------------------------------------------------------------
homeassistant/components/tellduslive/__init__.py          115     80    30%   58-79, 84-99, 104-117, 122-132, 140-146, 150-151, 155, 160-172, 176-195, 201-213, 219, 223
homeassistant/components/tellduslive/binary_sensor.py      18     18     0%   3-43
homeassistant/components/tellduslive/config_flow.py        73      0   100%
homeassistant/components/tellduslive/const.py              16      0   100%
homeassistant/components/tellduslive/cover.py              29     29     0%   3-59
homeassistant/components/tellduslive/entry.py              68     68     0%   3-119
homeassistant/components/tellduslive/light.py              43     43     0%   3-80
homeassistant/components/tellduslive/sensor.py             66     66     0%   3-197
homeassistant/components/tellduslive/switch.py             25     25     0%   3-53
-------------------------------------------------------------------------------------
TOTAL                                                     453    329    27%
```

Reply:
Let's correct it.

Reply:
Reverted